connector/CONFIG.md

```diff
@@ -133,20 +140,20 @@ Generates:
 - "apple tv"
 - "samsung tv"
 
-### External
+### External Sources
 
 - `external`: Array of indices you want to use.
-- [Optional] `external_source`: Accepts either `"source"` or `"target"` (See [Credentials](#credentials))
+- [Optional] `external_source`: Accepts either `"source"` or `"target"` (See [Credentials](#credentials)). ???If not specified, it uses the source of the main index.
```

Comment:
Actually, if not specified, it uses the app passed through the environment variables.
You can have up to three different apps for this connector.

1. The one it runs on: defined through`APPLICATION_ID` and `API_KEY` environment variables
2. The source app: where the main index and its analytics lies (defaults to 1.)
3. The target app: where the query suggestions index will be created (defaults to 1.)